requirements/extras/qpid.txt

```diff
@@ -1,2 +1,2 @@
-qpid-python>=0.26
-qpid-tools>=0.26
+python-qpid-proton
+qpid-tools
```

Comment:
i get this error while trying to install qpid-tools File "/tmp/pip-install-9104bo_x/qpid-python_a14df52a368846c0b0474e8c8647e96c/setup.py", line 42
        raise DistutilsFileError, \
                                ^
    SyntaxError: invalid syntax
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.